src/org/usfirst/frc/team2585/systems/WheelSystem.java

```diff
@@ -94,25 +128,71 @@ public double rampedForward() {
 		}
 		return forward;
 	}
+	
+	/**
+	 * @param rotation the input rotation value
+	 * @return the result of applying the exponent to the ROTATION_EXPONENT while maintaining its sign
+	 */
+	public double applyRotationExponent(double rotation) {
+		boolean negRotation = false;
+		if (rotation < 0) {
+			negRotation = true;
+		}
+		
+		rotation = Math.pow(Math.abs(rotation), ROTATION_EXPONENT);
+		if (negRotation) {
+			rotation *= -1;
+		}
+		return rotation;
+	}
+	
+	/**
+	 *  Update the values of the boost toggler and direction inversion toggler
+	 *  Then perform the appropriate actions based on the resulting togler states
+	 */	
+	private void checkTogglers() {
+		// Invert the drive train direction if necessary
+		invertDirectionToggler.toggle(input.invert());
+		boostToggler.toggle(input.boost());
+	}
+	
+	/**
+	 * Included so that the tests can ensure the gears are shifted appropriately
+	 * @return boolean denoting the state of the gear shifter solenoid
+	 */
+	public boolean getBoostState() {
+		return boostToggler.state();
+	}
+	
+	/**
+	 * Return the value of the input, but if its magnitude is less than the deadzone,
```

Comment:
Use the autogenerated javadoc comment for this. (@return). It likes weird for the return 0 to be on its own line, since that is where the annotation usually is.